kanidmd_web_ui/src/login.rs

```diff
@@ -20,6 +20,7 @@ extern "C" {
 pub struct LoginApp {
     link: ComponentLink<Self>,
     inputvalue: String,
+    #[allow(dead_code)]
```

Comment:
Shouldn't be needed, lstorage should be used in the latest commits.

Reply:
done